homework.test.js

```diff
@@ -28,5 +28,71 @@ async function homework(a, b, send, next) {
 }
 
 describe('Homework unit tests', () => {
-    //TODO: Implement unit tests that covers every edge case
-})
\ No newline at end of file
+        const a = 5
+        const b = 10
+        const send = jest.fn(x => x)
+        const next = jest.fn(x => x)
```

Comment:
Pierwsza ważna sprawa jest taka, że wartości początkowe dla testu powinniśmy deklarować per test. Każdy test jest oddzielną całością i musi mieć swoje wartości początkowe. Choćby dlatego, że jeśli współdzielisz pomiędzy testami mocki funkcji `send` i `next` to w każdym kolejnym teście będą się zmieniała liczba ich wywołań itd. To może być pierwszy powód dla którego coś Ci nie działa